bin/argo-api-authn-scripts/ams-create-users-cloud-info.py

```diff
@@ -0,0 +1,406 @@
+                if ams_usr_crt_req.status_code != 409:
+                    continue
+                else:
+                    user_exists = True
+                    # Get user by username
+                    ams_usr_get_req = requests.get(
+                        "https://" + ams_host + "/v1/users/" +
+                        user_binding_name + "?key=" + ams_token, verify=verify)
+                    LOGGER.info(ams_usr_get_req.text)
+                    req_data = json.loads(ams_usr_get_req.text)
+
+            # Create the respective AUTH binding
+            bd_data = {
+                'name': user_binding_name,
+                'service_uuid': authn_service_uuid,
+                'host': authn_service_host,
+                'dn': service_dn,
+                'unique_key': req_data["uuid"]
+            }
+            authn_binding_crt_req = requests.post(
+                "https://"+authn_host+"/v1/bindings?key="+authn_token,
+                data=json.dumps(bd_data), verify=verify)
+            LOGGER.info(authn_binding_crt_req.text)
+
+            if authn_binding_crt_req.status_code != 201:
+                LOGGER.critical(
+                    "Something went wrong while creating a binding." +
+                    "\nBody data: " + str(bd_data) + "\nResponse: " +
+                    authn_binding_crt_req.text)
+                if not user_exists:
+                    # delete the ams user, since binding could not be created
+                    requests.delete(
+                        "https://" + ams_host + "/v1/users/" + user_binding_name +
+                        "?key=" + ams_token, verify=verify)
+                continue
+
+            # count how many users have been created
+            user_count += 1
+
+            # create new topic
+            primary_key = service_endpoint. \
+                find("PRIMARY_KEY").text.replace(' ', '')
+            topic_name = 'SITE_' + sitename + '_ENDPOINT_' + primary_key
+            topic_crt_req = requests.put(
+                "https://" + ams_host + "/v1/projects/" + ams_project +
+                "/topics/" + topic_name + "?key=" + ams_token, verify=verify)
+
+            authorized_users = [user_binding_name]
+            if topic_crt_req.status_code != 200:
+                if topic_crt_req.status_code != 409:
+                    LOGGER.critical(
+                        "Something went wrong while creating topic " +
+                        topic_name + "\nResponse: " + topic_crt_req.text)
+                    continue
+                else:
+                    get_topic_acl_req = requests.get(
+                        "https://" + ams_host + "/v1/projects/" + ams_project +
+                        "/topics/" + topic_name + "?key=" + ams_token,
+                        verify=verify)
+                    if get_topic_acl_req.status_code == 200:
+                        acl_users = json.load(get_topic_acl_req.text)
```

Comment:
json.**load()** is used for file-like objects, and as a result, it produces the error
**AttributeError: 'unicode' object has no attribute 'read'**

use json.**loads()**